src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length);
+        return Color.values()[colorIndex].name();
+    }
+}
```

Comment:
```suggestion
public class ColorSupplier {
      private Random random = new Random();

    public String getRandomColor() {
        int colorIndex = random.nextInt(Color.values().length);
        return Color.values()[colorIndex].name();
    }
}
```

lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method